src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLBikeParkImpl.java

```diff
@@ -4,48 +4,55 @@
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 import org.opentripplanner.ext.legacygraphqlapi.generated.LegacyGraphQLDataFetchers;
-import org.opentripplanner.routing.bike_park.BikePark;
+import org.opentripplanner.routing.vehicle_parking.VehicleParking;
 
 public class LegacyGraphQLBikeParkImpl implements LegacyGraphQLDataFetchers.LegacyGraphQLBikePark {
+
     @Override
     public DataFetcher<Relay.ResolvedGlobalId> id() {
         return environment -> new Relay.ResolvedGlobalId(
                 "BikePark",
-                getSource(environment).id
+                getSource(environment).getId().toString()
         );
     }
 
     @Override
     public DataFetcher<String> bikeParkId() {
-        return environment -> getSource(environment).id;
+        return environment -> getSource(environment).getId().toString();
```

Comment:
I realized that maybe we should return the ids without the "feed id" in these legacy bikepark/carpark types like we have done with the BikeRentalStation vs VehicleRentalStation for backward compatibility.